monai/inferers/inferer.py

```diff
@@ -213,6 +213,7 @@ def __call__(
             device,
             self.progress,
             self.roi_weight_map,
+            None,
```

Comment:
May I know why adding this `None` value here?

Thanks.

Reply:
The `sliding_window_inferer` has an extra argument now (`process_fn`), which `SlidingWindowInferer` is not using it, so it should be set to None.

Reply:
I see, thanks.